lib/ddtrace/contrib/active_record/configuration/resolver.rb

```diff
@@ -9,6 +9,16 @@ module Configuration
         class Resolver < Contrib::Configuration::Resolver
           def initialize(configurations = nil)
             @configurations = configurations
+            @well_known_keys = {}
```

Comment:
This may be renamed to `@resolved_keys`.
Would you see any interest in using concurrent ruby data structures here?

Reply:
I like that name; I'll update, thanks!

As much as we'd like to, we can't use any libraries (including `concurrent-ruby`) as they may conflict with user's own copies of these gems. However if you have a custom resolver in your own application or integration, you're welcome to use it.